src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.tsx

```diff
@@ -427,6 +425,14 @@ export class EditDataSourceForm extends React.Component<EditDataSourceProps, Edi
         break;
 
       default:
+        const currentCredentials = (this.state.auth.credentials ?? {}) as {
+          [key: string]: string;
+        };
+        credentials = extractRegisteredAuthTypeCredentials(
+          currentCredentials,
+          this.state.auth.type,
+          this.authenticationMethodRegistery
```

Comment:
btw, `Registery` is typo

Reply:
@xinruiba Can you create an issue to track?

Reply:
And included the typo for crendentialFormField too? Thanks

Reply:
Issue created to track the `AuthenticationMethodRegistery` typo.
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6142

Thanks